docs/docs/lattices_crate/index.mdx

```diff
@@ -1,6 +1,9 @@
+import CrateDocs from '../../../lattices/README.md'
+
+
 # The `lattices` Crate
 
-{{#include ../lattices/README.md:book_include}}
```

Comment:
![image](https://github.com/hydro-project/hydroflow/assets/6778341/4f221128-4eb3-4f48-88d6-5354cee29683)

This part is not working quite right

Rustdoc lets you link to rust items without specifying the href https://docs.rs/lattices/latest/lattices/, but doesn't work in normal markdown (https://crates.io/crates/lattices and https://github.com/hydro-project/hydroflow/tree/main/lattices#lattices-1) ....

Reply:
I will fix this later...